### PR TITLE
docs: add v0.2.0 changelog

### PR DESCRIPTION
**Pull request by joeylichang**
### Description

Add v0.2.0 changelog

### Rationale

N/A

### Example

N/A

### Changes

Notable changes: 

N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## v0.2.0
+
+FEATURES
+* [\#358](https://github.com/bnb-chain/greenfield-storage-provider/pull/358) feat: sp services add pprof
+* [\#379](https://github.com/bnb-chain/greenfield-storage-provider/pull/379) feat:block syncer add read concurrency support
```
Comment (sysvm) [inline CHANGELOG.md:7]: Add empty space afrter `:`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+* [\#383](https://github.com/bnb-chain/greenfield-storage-provider/pull/383) feat: add universal endpoint view option
+* [\#389](https://github.com/bnb-chain/greenfield-storage-provider/pull/389) feat: signer async send sealObject tx
+* [\#402](https://github.com/bnb-chain/greenfield-storage-provider/pull/402) feat: bsdb switch
+* [\#404](https://github.com/bnb-chain/greenfield-storage-provider/pull/404) feat: list objects pagination & folder path
+* [\#406](https://github.com/bnb-chain/greenfield-storage-provider/pull/406) feat: adapt greenfield v0.47
+* [\#408](https://github.com/bnb-chain/greenfield-storage-provider/pull/408) feat: add gc worker
+* [\#410](https://github.com/bnb-chain/greenfield-storage-provider/pull/410) feat: support full-memory replicate task
+* [\#411](https://github.com/bnb-chain/greenfield-storage-provider/pull/411) feat:add upload download add bandwidth limit
```
Comment (sysvm) [inline CHANGELOG.md:15]: Add empty space after `:`

File: CHANGELOG.md (hunk 1 above)
Comment (joeylichang) [reply CHANGELOG.md:7]: it is commit content, change log should be same with the origin

File: CHANGELOG.md (hunk 2 above)
Comment (joeylichang) [reply CHANGELOG.md:16]: ditto

**Pull request merged**